pom.xml

```diff
@@ -36,6 +36,7 @@
     <module>nuxeo-ai-model</module>
     <module>nuxeo-ai-config</module>
     <module>nuxeo-ai-web-ui</module>
+    <module>nuxeo-ai-internal</module>
```

Comment:
This module no more needs a special treatment, remove the dedicated lines in tools/release.sh and tools/sprint.sh
https://github.com/nuxeo/nuxeo-ai/blob/766ddbd25b9f13f58c0c7ba931eea6440218b1f7/tools/release.sh#L43
https://github.com/nuxeo/nuxeo-ai/blob/766ddbd25b9f13f58c0c7ba931eea6440218b1f7/tools/release.sh#L58
https://github.com/nuxeo/nuxeo-ai/blob/766ddbd25b9f13f58c0c7ba931eea6440218b1f7/tools/sprint.sh#L128